rules.js

```diff
@@ -32,6 +32,9 @@
 	RULE_VUE: {
 		test: /\.vue$/,
 		loader: 'vue-loader',
+		options: {
+			prettify: false,
+		},
```

Comment:
I'd add a comment to describe why this is required and to not forget to remove it when the bug will be resolved.

```suggestion
		options: {
			// Vue 2.7 with Prettier 3 bug. See: https://github.com/vuejs/vue/issues/13052
			prettify: false,
		},
```